test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -513,7 +513,7 @@ private static void RunEndToEnd(MLContext mlContext, IDataView trainData, string
             // Construct the learning pipeline. Note that we are now providing a contract name for the custom mapping:
             // otherwise we will not be able to save the model.
             var estimator = mlContext.Transforms.CustomMapping<InputRow, OutputRow>(CustomMappings.IncomeMapping, nameof(CustomMappings.IncomeMapping))
-                .Append(mlContext.BinaryClassification.Trainers.FastTree(labelColumn: "Label"));
+                .Append(mlContext.BinaryClassification.Trainers.FastTree(labelColumnName: "Label"));
```

Comment:
This usually means the cookbook .md file needs to be updated as well.

Reply:
Thanks for making me take a second look at this.  Updated the cookbook .md file in a couple of places.

p.s. my initial search for "labelColumn" in the cookbook .md file did not return relevant results.  Apparently it used to be called just "label" at some point :)

---
In reply to: [259626158](https://github.com/dotnet/machinelearning/pull/2665#discussion_r259626158) [](ancestors = 259626158)